feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -573,13 +573,19 @@ class BillingWrapper(
                         }
                     }
                 }
-                BillingClient.BillingResponseCode.SERVICE_DISCONNECTED,
-                BillingClient.BillingResponseCode.USER_CANCELED,
+                BillingClient.BillingResponseCode.SERVICE_TIMEOUT,
+                BillingClient.BillingResponseCode.ERROR,
                 BillingClient.BillingResponseCode.SERVICE_UNAVAILABLE,
+                BillingClient.BillingResponseCode.USER_CANCELED,
```

Comment:
Hmm should we retry on a `USER_CANCELED` response code? Though I guess we shouldn't get this code in the `onBillingSetupFinished` callback and this is just to complete the cases, seems like it could be part of the codes where we don't retry? Just would like to understand why it was put here.